Documentation/configuration/metrics.rst

```diff
@@ -300,7 +300,7 @@ Kubernetes
 ======================================== ================================================== ========================================================
 Name                                     Labels                                             Description
 ======================================== ================================================== ========================================================
-``kubernetes_events_received_total``     ``scope``, ``action``, ``validity``, ``equiality`` Number of Kubernetes events received
+``kubernetes_events_received_total``     ``scope``, ``action``, ``validity``, ``equal``     Number of Kubernetes events received
```

Comment:
all other labels are noun, need your input if `equal` should be change to something like `equality` in implementation or not.

Reply:
No, the implementation is correct, only the docs need to be changed.